pykechain/client.py

```diff
@@ -3049,3 +3050,76 @@ def active_banner(self) -> Banner:
             data = active_banner_list[0]
 
         return Banner(json=data, client=self)
+
+    def expiring_download(self, *args, **kwargs) -> ExpiringDownload:
```

Comment:
one day we need to split this 3k++ file into subs :) -- the question is then 'when' :)

Reply:
Yes, it got pretty big. Maybe start an issue to at least have it somewhere?